frappe/commands.py

```diff
@@ -991,20 +991,64 @@ def drop_site(site, root_login='root', root_password=None):
 	from frappe.model.db_schema import DbManager
 	from frappe.utils.backups import scheduled_backup
 
+
 	frappe.init(site=site)
 	frappe.connect()
 	scheduled_backup(ignore_files=False, force=True)
 
+	# Archived site path must be outside the bench path, since with CENTRAL v2 after migrations path
```

Comment:
archived_sites path should be within frappe_bench by default. But, if we mention archived_sites in common_site_config.json, use that path!